archetypes/archetype-lambda/src/main/resources/global.vm

```diff
@@ -0,0 +1,29 @@
+## global variables used by the project
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+## TODO: there is no easy way to map service module artifactId to the client name derived the service name
+## As a workaround: mapping the common service here
+#if( $service == 'dynamodb')
+    #set ( $service = 'dynamoDb')
+#end
```

Comment:
If we want to invest time, we could generate these mappings and upload them to maven for the archetype to use.